examples/foood/yarn.lock

```diff
@@ -9,12 +9,40 @@
   dependencies:
     "@babel/highlight" "^7.10.4"
 
+"@babel/code-frame@^7.12.13":
```

Comment:
I don't think this lockfile should have changed -- can you revert this? (and the change to the package.json)